circuit_knitting/cutting/cut_wire_to_move.py

```diff
@@ -13,16 +13,34 @@
 """Function to transform a :class:`.CutWire` instruction to a :class:`.Move` instruction."""
 from __future__ import annotations
 
+from typing import Callable
 from itertools import groupby
 import numpy as np
```

Comment:
I normally lump imports by Python/External/CKT in lots of other places in the repo. Not a huge deal, but if you're indifferent...